batch_uploads_imageuploader.pl

```diff
@@ -63,8 +63,8 @@ =head2 Methods
 
 
 my $profile   = '';
-my $upload_id = undef; 
-my ($debug, $verbose) = (0,1);
+my $upload_id = undef;
+my ($debug, $verbose) = (0,0);
```

Comment:
why has this changed?

Reply:
Oh, maybe I should have proposed this change in a separate PR indeed.
The current line makes the imaging pipeline always verbose, no matter if the `--verbose` argument is provided or not. When calling the Python script from Perl, it becomes very noticeable since all database queries are logged.